<launch>
	<!-- run the autonomous_landing.cpp -->
	<node pkg="prometheus_demo" type="autonomous_landing" name="autonomous_landing" output="screen" launch-prefix="gnome-terminal --">
		<param name="uav_id" value="1" />
		<!-- 是否使用降落板绝对高度 -->
		<param name="use_pad_height" value="true" />
		<param name="pad_height" value="0.01" />
		<!-- 最大高度 -->
		<param name="max_height" value="10.0" />
		<!-- 相机安装偏差 -->
		<param name="camera_offset_x" value="0.0" />
		<param name="camera_offset_y" value="0.0" />
		<param name="camera_offset_z" value="-0.1" />
		<!-- 追踪控制参数 -->
		<param name="kpx_land" value="0.2" />
		<param name="kpy_land" value="0.2" />
		<param name="kpz_land" value="0.2" />
		<param name="arm_height_to_ground" value="0.3" />
		<param name="arm_distance_to_pad" value="0.2" />
	</node>
</launch>